generic/sysbench.py

```diff
@@ -64,9 +65,16 @@ def setUp(self):
                 self.urllink = self.params.get(
                     'url-link', default="https://github.com/akopytov/"
                                         "sysbench.git")
+                self.fixlink = self.params.get('fixlink', default=None)
                 self.bch = self.params.get('branch', default='master')
                 git.get_repo(self.urllink, branch=self.bch,
                              destination_dir=self.teststmpdir)
+
+                if self.fixlink:
+                    luajiturl = '%s/third_party/luajit/luajit/' % self.teststmpdir
```

Comment:
how about this:
```
self.fix_dir = self.params.get('fixdir', default=None)
...
if self.fixlink and self.fix_dir:
    fixdir = os.path.join(self.teststmpdir, self.fix_dir)
    shutil.rmtree(fixdir)
    git.get_repo(self.fixlink, branch="ppc64-port", destination_dir=lfixdir)
```

Reply:
@narasimhan-v `self.fix_dir` i think we need not to make it as variable as it is always constant and later operation depend on only this path so i think it should  be fine

Reply:
@PraveenPenguin When fix link comes from yaml, then fix dir also should be so..